.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
+  - 8
   - 5
   - 4
-  - 0.10
```

Comment:
The newest version of `mingo` uses `Symbol.Iterator`, which wasn't supported until Node `0.12`: http://node.green/#ES2015-built-ins-typed-arrays--TypedArray--prototype-Symbol-iterator-